src/app/frontend/deploy/createnamespace_controller.js

```diff
@@ -22,10 +22,11 @@ export default class NamespaceDialogController {
    * @param {!md.$dialog} $mdDialog
    * @param {!angular.$log} $log
    * @param {!angular.$resource} $resource
+   * @param {!Object} errorDialog
```

Comment:
Add a todo with reference to the issue. We will fix it in a separate track. We just need to make it easy to track places to fix.